iam/policy-templates/ci-cd.json

```diff
@@ -103,8 +103,9 @@
       ],
       "Resource": [
         "arn:aws:lambda:*:$account_id:function:dss-*",
-        "arn:aws:es:*:$account_id:domain/dss-index-dev",
-        "arn:aws:es:*:$account_id:domain/dss-index-dev/*",
+        "arn:aws:es:*:$account_id:domain/dss-index-dev", "arn:aws:es:*:$account_id:domain/dss-index-dev/*",
+        "arn:aws:es:*:$account_id:domain/dss-index-integration", "arn:aws:es:*:$account_id:domain/dss-index-integration/*",
+        "arn:aws:es:*:$account_id:domain/dss-index-staging", "arn:aws:es:*:$account_id:domain/dss-index-staging/*",
```

Comment:
Is this line correct, given that staging uses the dev ES domain?

Reply:
I've done this in anticipation of a remedy soon to come.

The [commit message](https://github.com/HumanCellAtlas/data-store/pull/1029/commits/2775f0ccd652bb4b9bc73f4bddd89ab1b07a6d87) mentions this too.